pluribus/games/short_deck/state.py

```diff
@@ -219,7 +219,7 @@ def info_set(self) -> str:
         eval_cards = tuple([card.eval_card for card in cards])
```

Comment:
would you mind adding a new property to this class:
```python
    @property
    def players(self) -> List[ShortDeckPokerPlayer]:
        """Returns players in table."""
        return self._table.players
``` 

This is supporting a comment I made in the following python script

Reply:
Sounds good!